Date: Thu Mar  3 14:37:16 CET 2005
From: Vojtech Pavlik <vojtech@suse.cz>
Subject: Fix a typo in i8042 ACPI printk()
=====================================================================================


ChangeSet@1.2068, 2005-03-03 14:26:48+01:00, vojtech@suse.cz
  input: Fix a typo in i8042 ACPI printk().
  
  Signed-off-by: Vojtech Pavlik <vojtech@suse.cz>


 i8042-x86ia64io.h |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)


diff -Nru a/drivers/input/serio/i8042-x86ia64io.h b/drivers/input/serio/i8042-x86ia64io.h
--- a/drivers/input/serio/i8042-x86ia64io.h	2005-03-03 14:36:39 +01:00
+++ b/drivers/input/serio/i8042-x86ia64io.h	2005-03-03 14:36:39 +01:00
@@ -241,7 +241,7 @@
 	i8042_aux_irq = i8042_pnp_aux_irq;
 
 	printk(KERN_INFO "PNP: PS/2 Controller [%s%s%s] at %#x,%#x irq %d%s%d\n",
-		i8042_pnp_kbd_name, (result_kbd > 0 || result_aux > 0) ? "," : "", i8042_pnp_aux_name,
+		i8042_pnp_kbd_name, (result_kbd > 0 && result_aux > 0) ? "," : "", i8042_pnp_aux_name,
 		i8042_data_reg, i8042_command_reg, i8042_kbd_irq,
 		(result_aux > 0) ? "," : "", i8042_aux_irq);
 
